src/main/scala/com/advancedtelematic/director/data/DataType.scala

```diff
@@ -59,6 +59,20 @@ object DataType {
 
   final case class EcuTarget(namespace: Namespace, version: Int, ecuIdentifier: EcuIdentifier, customImage: CustomImage)
 
+  final case class EcuUpdateAssignment(namespace: Namespace,
+                                   deviceId: DeviceId,
+                                   ecuIdentifier: EcuIdentifier,
+                                   version: Int,
+                                   customImage: CustomImage)
+
+  final case class DeviceUpdateAssignment(
+    namespace: Namespace,
+    deviceId: DeviceId,
+    correlationId: Option[CorrelationId],
+    updateId: Option[UpdateId],
+    version: Int,
+    inFlight: Boolean)
```

Comment:
Why not use the same field name than in the table here? IMO `served` sounds more understandable.